managed/models/database.go

```diff
@@ -1173,17 +1274,21 @@ func migrateDB(db *reform.DB, params SetupDBParams) error {
 			return err
 		}
 
-		if err = setupFixture1(tx.Querier, params); err != nil {
+		err = EncryptDB(tx, params, itemsToEncrypt)
+		if err != nil {
 			return err
 		}
-		if err = setupFixture2(tx.Querier, params.Username, params.Password); err != nil {
+
+		err = setupPMMServerAgents(tx.Querier, params)
+		if err != nil {
 			return err
 		}
```

Comment:
why are we doing it after encryption?

Reply:
setupPMMServerAgents using CreateAgent methods. Those methods will encrypt agent during adding. Then EncryptDB is called and it will end up with double encryption for PMM server agents, since they are already encrypted once during CreateAgent.

With changed order first all current agents are encrypted, then PMM server ones are added. So non of them is double encrypted.

Reply:
got it, thanks